src/antithesis_sdk/_internal/handlers.py

```diff
@@ -8,13 +8,18 @@
 from abc import ABC, abstractmethod
 import cffi  # type: ignore[import-untyped]
 from io import TextIOWrapper
+import json
 import os
 import random
+import sys
 from typing import Optional
 
+from .sdk_constants import (
+    LOCAL_OUTPUT_ENV_VAR,
+    ANTITHESIS_SDK_VERSION,
+    ANTITHESIS_PROTOCOL_VERSION,
+)
 
-LOCAL_OUTPUT_ENV_VAR: str = "ANTITHESIS_SDK_LOCAL_OUTPUT"
-ASSERTION_CATALOG_ENV_VAR: str = "ANTITHESIS_ASSERTION_CATALOG"
 VOIDSTAR_PATH = "/usr/lib/libvoidstar.so"
```

Comment:
Should this also be in consts?

Reply:
Updating `VOIDSTAR_PATH` to `_VOIDSTAR_PATH` - it should be considered private, and not to be used outside of `'handlers.py'`.  Also updated `CDEF_VOIDSTAR` to `_CDEF_VOIDSTAR` using the same reasoning.